src/extension/background-script/actions/ln/transactions.ts

```diff
@@ -0,0 +1,45 @@
+import utils from "~/common/lib/utils";
+import state from "~/extension/background-script/state";
+import type { Invoice, MessageTransactions } from "~/types";
+
+const transactions = async (message: MessageTransactions) => {
+  const isSettled = message.args.isSettled;
+  const limit = message.args.limit;
```

Comment:
FYI: This is quite messed up implementation here. 

I think the limit / offset would need to be passed from the UI all the way down to the  connector.getTransactions() call since the connector is the one loading the actual transactions. 

Just to keep in mind when we want to add pagination for transactions one day.

Reply:
we use the same hook, for showing all transactions and for showing limited transactions, so we do need to pass  according to context.

Reply:
we are not passing it to connector, connector will return all transactions by default !

we are doing slicing in the action itself
```
    if (limit) {
      transactions = transactions.slice(0, limit);
    }
```